tutorials/advanced_features/Virtual Devices.ipynb

```diff
@@ -79,40 +55,32 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Converting a Device in a Virtual Device"
+    "### Converting a Device into a Virtual Device"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "However sometimes we want to perform the computations on a more permissive device, a device that would have more dimensions or more atoms for instance, or more types of channels. This can be done on an emulator behaving like a device. The `VirtualDevice` class is useful to define such an emulator, a virtual device. Let's start by configuring a virtual device having the same parameters as `Chadoq2`. To do this, we use the `Device.to_virtual()` method that creates a virtual device from a physical one. "
+    "However we sometimes want to perform the computations on a more permissive device, a device that would have more dimensions or more atoms for instance, or more types of channels. This can be done on an emulator behaving like a device. The `VirtualDevice` class is useful to define such an emulator, a virtual device. Let's start by configuring a virtual device having the same parameters as `Chadoq2`. To do this, we use the `Device.to_virtual()` method that creates a virtual device from a physical one. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "VirtualDevice(name='Chadoq2', dimensions=2, rydberg_level=70, min_atom_distance=4, max_atom_num=100, max_radial_distance=50, interaction_coeff_xy=None, supports_slm_mask=True, max_layout_filling=0.5, reusable_channels=False, channel_ids=('rydberg_global', 'rydberg_local', 'raman_local'), channel_objects=(Rydberg.Global(Max Absolute Detuning: 125.66370614359172 rad/µs, Max Amplitude: 15.707963267948966 rad/µs, Clock period: 4 ns, Minimum pulse duration: 16 ns, Maximum pulse duration: 67108864 ns, Basis: 'ground-rydberg'), Rydberg.Local(Max Absolute Detuning: 125.66370614359172 rad/µs, Max Amplitude: 62.83185307179586 rad/µs, Minimum retarget time: 220 ns, Fixed retarget time: 0 ns, Max targets: 1, Clock period: 4 ns, Minimum pulse duration: 16 ns, Maximum pulse duration: 67108864 ns, Basis: 'ground-rydberg'), Raman.Local(Max Absolute Detuning: 125.66370614359172 rad/µs, Max Amplitude: 62.83185307179586 rad/µs, Minimum retarget time: 220 ns, Fixed retarget time: 0 ns, Max targets: 1, Clock period: 4 ns, Minimum pulse duration: 16 ns, Maximum pulse duration: 67108864 ns, Basis: 'digital')), _channels=())\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# Converting the Device object in a VirtualDevice object\n",
     "VirtualChadoq = Chadoq2.to_virtual()\n",
-    "print(VirtualChadoq)"
+    "VirtualChadoq.__repr__()"
```

Comment:
Why this change? I'm okay with using the `repr`, but in that case I would prefer `repr(VirtualChadoq)` instead